README.md

```diff
@@ -1,3 +1,37 @@
-# harvester-prototype
 
-This repository is for the frontend of the project react
+# Harvester Prototype (FertiScan)
+
+## Overview Description
+---
+The Harvester project, represents a significant 
+advancement in fertilizer bag inspection. The primary goal of this prototype is
+to test and validate various potential solutions that could be integrated into 
+the final project. The AI-Lab team, utilizing unit tests, strives to determine 
+the best approach to address issues associated with fertilizer bag inspection.
+
+## Key Features
+---
+**Gemini Vertex API:** An application programming interface designed to 
+simulate human conversation, capable of answering questions and processing 
+a large number of images.
+**Testing Process:** Rigorous testing is conducted to demonstrate that
+the Gemini Vertex API, created by Google, is the appropriate method for the 
+Harvester project. 
+It meets various criteria and achieves two main objectives: 
+reading labels during recording and identifying differences with bags analyzed 
+by inspectors.
+
+## Potential Advantages
+---
+**Increased Efficiency:** By automating fertilizer bag inspection, the project 
+aims to increase the efficiency and accuracy of the inspection process.
+**Advanced Analysis:** Through the integration of the Gemini Vertex API, the 
+system can analyze fertilizer bag labels more thoroughly, identifying not only 
+basic information but also subtle differences that may escape human inspection.
+
+## Future Outlook
+The Harvester prototype is a crucial step toward realizing a
+comprehensive automated inspection system. With continuous testing and 
+improvements, it has the potential to transform the way fertilizer bag 
+inspections are conducted, bringing an advanced technological solution 
+to the agricultural sector.
```

Comment:
You have a lot of trailing markdown lint error that can be solve with a parameter in vscode that will trim all whitespace when you save a document.

DO `CTRL + ,` then go to Text Editor>Files and find these Trim settings and you will be good to go

![image](https://github.com/ai-cfia/harvester-prototype/assets/96267006/2109d843-4d6b-402a-9092-13a46b34636e)